.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+name: Master CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master    
```

Comment:
I think it is not triggering because there is no such action in master, therefore github is smart enough to avoid triggering the first repo action from a non-owner (since it might contain malicious code).

Reply:
The reason I said that was because I can't see an "approve" or "run" button. We'll have to merge it then without testing during the PR. I'll do that once the Arm target is set for runs-on